tests/mongo/fields/test_crypto.py

```diff
@@ -20,34 +22,34 @@ class Secret(Document):
         col = connection._get_db().secret
 
         # Test creating password
-        s = Secret.objects.create(password='hello')
-        self.assertEqual(s.password, 'hello')
+        s = Secret.objects.create(password=b'hello')
```

Comment:
Can't the field itself do the UTF-8  encoding/decoding from/to unicode? I wouldn't want to have to remember to do this myself everywhere I use this field.

Reply:
I won't _decode_ it since there will be binary data (keys/passwords/etc) but encoding it to bytes if you pass a unicode in makes sense for convenience. Implemented that.